src/callbacks.jl

```diff
@@ -8,6 +8,25 @@
 # See https://github.com/jump-dev/JuMP.jl
 #############################################################################
 
+"""
+    callback_node_status(cb_data, model::Model)
+
+Return an [`MOI.CallbackNodeStatusCode`](@ref) enum, indicating if the current
+primal solution available from [`callback_value`](@ref) is integer feasible.
+"""
+function callback_node_status(cb_data, model::Model)
+    # TODO(odow):
+    # MOI defines `is_set_by_optimize(::CallbackNodeStatus) = true`.
+    # This causes problems for JuMP because it checks the termination_status to
+    # see if optimize! has been called. Solutions are:
+    # 1) defining is_set_by_optimize = false
+    # 2) adding a flag to JuMP to store whether it is in a callback
+    # 3) adding IN_OPTIMIZE to termination_status for callbacks
+    # Once this is resolved, we can replace the current function with:
+    #     MOI.get(owner_model(x), MOI.CallbackVariablePrimal(cb_data), x)
```

Comment:
The comment should be updated to `MOI.get(model, MOI.CallbackNodeStatus(cb_data))`.